bluegreen_deploy.py

```diff
@@ -267,31 +264,45 @@ def swap_bluegreen_apps(args, new_app, old_app, timestamp):
         if waiting_for_drained_listeners(listeners):
             continue
 
-        drained_task_ids = \
-            find_drained_task_ids(old_app, listeners, haproxy_count)
+        return find_drained_task_ids(old_app, listeners, haproxy_count)
+
+    logger.info('Timed out waiting for drained tasks, Continuing...')
+    return []
```

Comment:
As per my other comment, we should just return all the backends marked for draining at this point.